### PR TITLE
OSDOCS-11367-main:fixes live migration support

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.16+

Issue:
https://issues.redhat.com/browse/OSDOCS-11367

Link to docs preview:
https://79026--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html#nw-ovn-kubernetes-live-migration-about_migrate-from-openshift-sdn

QE review:
- [ ] QE has approved this change.

Additional information:
https://github.com/openshift/openshift-docs/pull/79013

### CODE REVIEW

File: modules/nw-ovn-kubernetes-live-migration-about.adoc [hunk 1]
```diff
@@ -5,7 +5,9 @@
...
 = Limited live migration to the OVN-Kubernetes network plugin overview
 
-The limited live migration method is the process in which the OpenShift SDN network plugin and its network configurations, connections, and associated resources, are migrated to the OVN-Kubernetes network plugin without service interruption. It is available for {product-title}, {product-dedicated}, {product-rosa}, and Azure Red Hat OpenShift deployment types. It is not available for HyperShift deployment types. This migration method is valuable for deployment types that require constant service availability and offers the following benefits:
+The limited live migration method is the process in which the OpenShift SDN network plugin and its network configurations, connections, and associated resources, are migrated to the OVN-Kubernetes network plugin without service interruption. It is available for {product-title}.
+// Azure Red Hat OpenShift deployment types will needed added in .z stream.
+It is not available for HyperShift deployment types. This migration method is valuable for deployment types that require constant service availability and offers the following benefits:
```
Comment (lahinson) [inline modules/nw-ovn-kubernetes-live-migration-about.adoc:10]:
```suggestion
It is not available for hosted control plane deployment types. This migration method is valuable for deployment types that require constant service availability and offers the following benefits:
```

**Review by lahinson — COMMENTED** [review]
Same comment here about HyperShift terminology. Otherwise, it looks good!

File: modules/nw-ovn-kubernetes-live-migration-about.adoc [hunk 2]
```diff
@@ -5,7 +5,9 @@
...
 = Limited live migration to the OVN-Kubernetes network plugin overview
 
-The limited live migration method is the process in which the OpenShift SDN network plugin and its network configurations, connections, and associated resources, are migrated to the OVN-Kubernetes network plugin without service interruption. It is available for {product-title}, {product-dedicated}, {product-rosa}, and Azure Red Hat OpenShift deployment types. It is not available for HyperShift deployment types. This migration method is valuable for deployment types that require constant service availability and offers the following benefits:
+The limited live migration method is the process in which the OpenShift SDN network plugin and its network configurations, connections, and associated resources, are migrated to the OVN-Kubernetes network plugin without service interruption. It is available for {product-title}.
+// Azure Red Hat OpenShift deployment types will needed added in .z stream.
+It is not available for hosted control plane deployment types. This migration method is valuable for deployment types that require constant service availability and offers the following benefits:
```
Comment (weliang1) [inline modules/nw-ovn-kubernetes-live-migration-about.adoc:10]: In 4.16, there is no AliCloud IPI cluster for QE to test, please remove IPI

**Pull request merged**